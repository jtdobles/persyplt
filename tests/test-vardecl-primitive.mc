void main(){
	int a = -8;
	printi(a);

	int aa = 4;
	printi(aa);
	
	float b = 1.2;
	printf(b);

	bool c = true;
	bool d = false;
	prints(c);

	string s = "Hello World";
	prints(s);
}