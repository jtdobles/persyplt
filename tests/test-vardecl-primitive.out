-8
1.200000
FAILED
  ./test-vardecl-primitive.exe failed on ./test-vardecl-primitive.exe
